src/v3/src/types/context.ts

```diff
@@ -42,6 +42,8 @@ export type IWidgetContext = {
   loading: boolean;
   setLoading: StateUpdater<boolean>;
   setWidgetRendered: StateUpdater<boolean>;
+  loginHint?: string;
+  setLoginHint: StateUpdater<string | undefined>;
```

Comment:
Is there a way to avoid adding these to the widget context? I'm not sure if we need state variables to hold the `loginHint`. I'm thinking it can potentially be pushed out to the transformers and we only use the `widgetContext` to get the feature flag info.

Reply:
The problem was we needed a way for the loginHint/username to persist across views.  Since the `loginHint` is grabbed from the data in the identifier input field, we also need to be able to access it in the next flow with the Open Okta Verify button.